inc/saiport.h

```diff
@@ -438,7 +438,7 @@ typedef enum _sai_port_attr_t
      * @brief Media Type
      *
      * @type sai_port_media_type_t
-     * @flags CREATE_ONLY
+     * @flags CREATE_AND_SET
```

Comment:
I think the argument is that SFPs are modular.  You could swap out fiber & copper based SFPs dynamically.